sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,179 @@
+from sympy.physics.mechanics import KanesMethod, Body
+
+__all__ = ['JointsMethod']
+
+
+class JointsMethod(object):
+    """Method for formulating the equations of motion using a set of interconnected bodies with joints.
+
+    Parameters
+    ==========
+
+    body : Body or ReferenceFrame
+        The inertial(Newtonion) frame.
+    *joints : Joint
+        The joints in the system
+
+    Attributes
+    ==========
+
+    q, u : Matrix
+        Matrices of the generalized coordinates and speeds
+    bodylist : iterable
+        Iterable of Body objects in the system.
+    forcelist : iterable
+        Iterable of (Point, vector) or (ReferenceFrame, vector) tuples
+        describing the forces on the system.
+    mass_matrix : Matrix
+        The system's mass matrix
+    forcing : Matrix
+        The system's forcing vector
+    mass_matrix_full : Matrix
+        The "mass matrix" for the u's and q's
+    forcing_full : Matrix
+        The "forcing vector" for the u's and q's
+    method : KanesMethod or Lagrange's method
+        Method's object.
+
+    Examples
+    ========
+
+    This is a simple example for a one degree of freedom translational
+    spring-mass-damper.
+
+    >>> from sympy import symbols
+    >>> from sympy.physics.mechanics import Body, JointsMethod, PrismaticJoint
+    >>> from sympy.physics.vector import dynamicsymbols
+    >>> c, k = symbols('c k')
+    >>> q, u = dynamicsymbols('q u')
+    >>> W = Body('W')
+    >>> B = Body('B')
+    >>> J = PrismaticJoint('J', W, B, coordinates=q, speeds=u)
+    >>> W.apply_force(c*u*W.x, reaction_body=B)
+    >>> W.apply_force(k*q*W.x, reaction_body=B)
+    >>> method = JointsMethod(W, J)
+    >>> (fr, frstar) = method.form_eoms()
+    >>> MM = method.mass_matrix
+    >>> forcing = method.forcing
+    >>> rhs = MM.inv() * forcing
```

Comment:
```suggestion
    >>> rhs = MM.LUSolve(forcing)
```

Using a linear system solver is what we should be teaching in the docs.

Reply:
It gives error.
```
AttributeError: 'MutableDenseMatrix' object has no attribute 'LUSolve'
```

Reply:
Check the documentation for matrices then.